src/htmlhelp.cpp

```diff
@@ -134,7 +134,17 @@ static QCString field2URL(const IndexField *f,bool checkReversed)
   QCString result = f->url + Doxygen::htmlFileExtension;
   if (!f->anchor.isEmpty() && (!checkReversed || f->reversed)) 
   {
-    result+="#"+f->anchor;  
+    result+="#";
```

Comment:
Sounds reasonable; I'm not too familiar with the Doxygen code, so wasn't aware this function even existed. I'll test it and update the pull request accordingly.